app/webpacker/components/RegistrationsV2/Register/PaymentStep.jsx

```diff
@@ -86,6 +88,14 @@ export default function PaymentStep({
     setIsLoading(false);
   };
 
+  if (!competitionInfo['registration_currently_open?']) {
+    return (
+      <Message color="red">
+        <I18nHTMLTranslate i18nKey="registrations.payment_form.errors.registration_closed" />
```

Comment:
You don't need the component if you don't inject any HTML fragments. Your new i18n key is just a plain and simple string without any replacements, so `I18n.t('foo.bar')` will do

Reply:
Fixed!